source/app/app.js

```diff
@@ -46,4 +47,165 @@ app.on('window-all-closed', function() {
 app.on('ready', function() {
    // Show intro
    windowManager.buildWindowOfType("intro");
+
+   // Show standard menu if on OS X
+   if (process.platform == 'darwin') {
+     var template = [
+       {
+         label: name,
```

Comment:
This one throws an error, have you tried it?
![image](https://cloud.githubusercontent.com/assets/768052/12728307/e02e3000-c929-11e5-9fcd-9eb9551caa71.png)